x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/escalation_uac_sdclt.json

```diff
@@ -0,0 +1,41 @@
+{
+  "author": [
+    "Elastic"
+  ],
+  "description": "Identifies User Account Control (UAC) bypass via sdclt.exe. Attackers bypass UAC to stealthily execute code with elevated permissions.",
+  "from": "now-9m",
+  "index": [
+    "logs-endpoint.events.*",
+    "winlogbeat-*"
+  ],
+  "language": "eql",
+  "license": "Elastic License",
+  "name": "Bypass UAC via Sdclt",
+  "query": "sequence with maxspan=1m\n  [process where event.type in (\"start\", \"process_started\") and process.name == \"sdclt.exe\" and\n     /* process.code_signature.* fields need to be populated for 7.10 */\n     process.code_signature.subject_name == \"Microsoft Corporation\" and process.code_signature.trusted == true and\n     process.args == \"/kickoffelev\"\n  ] by process.entity_id\n  [process where event.type in (\"start\", \"process_started\") and process.parent.name == \"sdclt.exe\" and\n     process.executable not in (\"C:\\\\Windows\\\\System32\\\\sdclt.exe\",\n                                \"C:\\\\Windows\\\\System32\\\\control.exe\",\n                                \"C:\\\\Windows\\\\SysWOW64\\\\sdclt.exe\",\n                                \"C:\\\\Windows\\\\SysWOW64\\\\control.exe\")\n  ] by process.parent.entity_id\n",
```

Comment:
Just commenting for posterity that we'll remove `/* process.code_signature.* fields need to be populated for 7.10 */` in a follow-up PR.